code/game/machinery/vending.dm

```diff
@@ -1051,7 +1051,8 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/clothing/mask/joy = 1,
 					/obj/item/clothing/head/cueball = 1,
 					/obj/item/clothing/under/scratch = 1,
-        			/obj/item/clothing/under/sailor = 1)
+        			/obj/item/clothing/under/sailor = 1,
+        			/obj/item/clothing/ears/headphones = 8)

```

Comment:
Autodrobe is theater access.
Clothesmate is public access. Starts around line 1150.